web-api/src/personalization/filter.rs

```diff
@@ -185,7 +185,7 @@ pub(crate) enum CombineOp {
 pub(crate) struct Filters(Vec<Filter>);
 
 impl Filters {
-    const MAX_LEN: usize = 10;
+    const MAX_LEN: usize = 500;
```

Comment:
not sure if this is the right value 
in the pr in which we implement the `in` filter we don't touch that value but instead have the value defined here https://github.com/xaynetwork/xayn_discovery_engine/blob/2a7f03e88083c03b920a8f5cf242366923dc81cd/web-api/src/personalization/filter.rs#L99